flask_appbuilder/security/manager.py

```diff
@@ -600,12 +600,13 @@ def get_oauth_user_info(self, provider, resp):
             me = self._azure_jwt_token_parse(id_token)
             log.debug("Parse JWT token : {0}".format(me))
             return {
-                "name": me["name"],
+                "name": me.get("name", ""),
                 "email": me["upn"],
-                "first_name": me["given_name"],
-                "last_name": me["family_name"],
+                "first_name": me.get("given_name", ""),
+                "last_name": me.get("family_name", ""),
                 "id": me["oid"],
                 "username": me["oid"],
+                "roles": me.get("roles", []),
```

Comment:
Why the roles here?

Reply:
I thought it can be used later like customized security manager, and I found my thought is wrong.
I will delete the line